themes/cerberus/index.txt

```diff
@@ -1,31 +1,39 @@
-<%- title %>
-
+<%- title %><br />
+<br />
 <% if (locals.intro) { %>
 <% intro.forEach(function (introItem) { -%>
-<%- introItem %>
+<%- introItem %><br />
 <% }) -%>
 <% } %>
-
+<% if (locals.intro) { %>
```

Comment:
The additional if statement here is redundant. Take a look at this code:

```js
<% if (locals.intro) { %>
<% intro.forEach(function (introItem) { -%>
<%- introItem %><br />
<% }) -%>
<% } %>
<% if (locals.intro) { %>
<br />
<% } %>
```

You could refactor it as follows:
```js
<% if (locals.intro) { %>
<% intro.forEach(function (introItem) { -%>
<%- introItem %><br />
<% }) -%>
<br />
<% } %>
```

All I did was remove the following:
```js

<% } %>
<% if (locals.intro) { %>
```